app/static/landing-page.css

```diff
@@ -57,10 +57,12 @@ header {
 }
 
 .right a:hover {
-  background-color: #6070ff;
+  /* background-color: #6070ff;
   color: #fff;
-  border-radius: 5px;
+  border-radius: 5px; */
```

Comment:
- [ ] Please do not comment code. This can make your code messy. Take a look at the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).